src/haplotypes/model.rs

```diff
@@ -127,37 +132,16 @@ impl Likelihood {
         // TODO compute likelihood based on Varlociraptor VAFs.
         // Let us postpone this until we have a working version with kallisto only.
         let variant_matrix: Vec<BitVec> = data.haplotype_variants.values().cloned().collect();
-        let likelihood = Prob(1.0);
-        for (variant, bv) in data.haplotype_variants.iter() {
+        let mut likelihood = Prob(1.0);
+        for (bv, (_, afd)) in variant_matrix.iter().zip(data.haplotype_calls.iter()) {
```

Comment:
`bv` is not a good name. The name should rather tell something about the meaning, not the type of the variable.

Reply:
Also, instead of having a for loop here, it is more idiomatic to use something like
```
let likelihood = variant_matrix.iter().zip(data.haplotype_calls.iter()).map(| ... | {
    ...
}).sum()
```